### ISSUE TITLE
Undefined index: provider

### ISSUE THREAD

**Issue by nurmanhabib** [laravel/lumen-framework#372]
Fresh install Lumen 5.2

```
$app->register(App\Providers\AppServiceProvider::class);
$app->register(App\Providers\AuthServiceProvider::class);
$app->register(App\Providers\EventServiceProvider::class);
```

Database is OK

![screenshot_9](https://cloud.githubusercontent.com/assets/5308494/13285843/c4001fc0-db2f-11e5-88a7-a9d726acb1dc.png)

Whats wrong?

**Comment by GrahamCampbell** [comment]
Looks like your config file is for lumen 5.1 but you have lumen 5.2.

**Comment by GrahamCampbell** [comment]
> Fresh install Lumen 5.2

Errr. Ping @taylorotwell.

**Comment by Garbee** [comment]
You have to change `config/auth` to have the provider as `token` instead of `api`. The offending commit that caused this was reverted in #367. Taylor has yet to tag another release of the framework to get the fix out. Once a new tag is done, an update will then break it again if you change your user-config. So do something, but make sure your config and application registration of the auth method are the same.

Simplest thing is, just pull in the auth config to your repo then change token to api. Then once the fix is in, you can simply delete the config if you aren't modifying anything else and be done.

**Comment by sagarrabadiya** [comment]
seems not working with the updated file also, changed driver to api, its working as application in browser, but when try to run phpunit testcase with actingAs() its going in loop check the screen shot below

-- test case
![image](https://cloud.githubusercontent.com/assets/5405403/13742838/455ad064-ea04-11e5-8d69-edc1614654ce.png)

-- terminal output
![image](https://cloud.githubusercontent.com/assets/5405403/13742863/6723e5aa-ea04-11e5-888f-d5b3bfcb77bc.png)

**Comment by yuloh** [comment]
Can this issue be closed @GrahamCampbell ?

The fix for @nurmanhabib's issue was fixed in 5.2.6.

@sagarrabadiya I tried to reproduce your issue on a fresh install of Lumen and everything worked fine.  If you are still having problems you should probably open up a separate issue.  I used these steps:

1. Uncomment all service providers in `bootstrap/app.php`.
2. Add this to `routes.php`:

```php
$app->get('/', function () use ($app) {
    return json_encode(['hello' => 'world']);
});
```

3. Add this to `ExampleTest.php`:

```php
$this
            ->actingAs(new App\User())
            ->get('/')
            ->seeJson(['hello' => 'world']);
```

**Comment by sagarrabadiya** [comment]
@yuloh yes i also updated lumen to latest version and its working now so this can be closed now
